src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -254,7 +254,7 @@ public Default(final String... users) {
                     throw new IllegalStateException(ex);
                 }
             } else {
-                identity = new Opt.Empty<>();
+                identity = new Optional<>(null);
```

Comment:
@Happy-Neko should be `Opt.Empty` (please see https://en.wikipedia.org/wiki/Null_Object_pattern)